### PR TITLE
ncm-opennebula: Include OpenNebula labels support

### LINKED ISSUE #959: ncm-opennebula: Include Labels support
This is a nice feature included since ONE 5.x, now is possible to include labels or tags to any resource (templates, running VMs, vnets...):

http://opennebula.org/opennebula-5-0-features-labels/

Only requires to include LABEL variable within the ONE resource template, labels are included in the database and it is also possible to make queries from the frontend.

### LINKED ISSUE #966: ncm-opennebula: src/test/perl/host.t fails with Net::OpenNebula 0.309.0
I've been trying to figure out why this is happening now when it didn't happen for RC1.
Perhaps I've managed to mess up my build environment again, but as always I start with a clean VM each time.

The verbose log for the relevant test is [in this gist](https://gist.github.com/jrha/102fb31d6f8fb233fb458bb46069e42a).

Any ideas?

### PR DESCRIPTION
**Pull request by alvarosimon**
**NOTE:** This PR requires `Net-OpenNebula-0.309.0` to work

* Fix #959 : Include labels support
* Fix #966 : fix `Net::OpenNebula` `0.309.0`  host rpc_history

### CODE REVIEW

**Comment by alvarosimon** [comment]
Tested in altaria, I didn't get issues (and this time I was using `0.309.0` version :smirk:).

File: ncm-opennebula/src/main/resources/tests/regexps/user_one/user_forbidden [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Opennebula test user forbidden
 ---
+multiline
```
Comment (stdweird) [inline ncm-opennebula/src/main/resources/tests/regexps/user_one/user_forbidden]: this is default

**Comment by stdweird** [comment]
@jrha this requires new perl-Net-OpenNebula rpm
you can make these yourself with
```
cd /tmp
yum install -y cpanspec
cpanspec -b --force --packager mynameisjames Net::OpenNebula 
```
(same for `Net::FreeIPA` and `File::AtomicWrite` which will be required soon)

i can also mail you the rpms :smile:

**Comment by stdweird** [comment]
retest this please

**Comment by stdweird** [comment]
@jrha if you want this in 16.10 to avoid the failing unittest, feel free to change the milestone

**Comment by jrha** [comment]
The failing test isn't an issue for me now, so I'd rather leave this for 16.12.

File: ncm-opennebula/src/main/resources/tests/regexps/user_one/user_forbidden (hunk 1 above)
Comment (alvarosimon) [inline ncm-opennebula/src/main/resources/tests/regexps/user_one/user_forbidden]: removed

**Pull request merged**